community/SDK-GOVERNANCE.md

```diff
@@ -8,6 +8,27 @@ The community is organized as follows:
   [cloudevents/sdk-go project](https://github.com/cloudevents/sdk-go)
 - The union of all the `sdk-*-maintainers` assembles the _sdk maintainers_ group
 
+## New SDKs
+
+To propose a new SDK for the community, a PR should be opened in the `spec`
+repository with the documentation changes necessary to point to the new repo.
+For example, changes to the `README.md` and `SDK.md` files. At that time the
```

Comment:
Can we add a sentence here about the "usefulness" of the new sdk, like "new sdk should integrate with a technology stack that actual sdk portfolio doesn't reach", or something like that

Reply:
done - see what you think

Reply:
Ok for me